packages/mail/integration.jest.js

```diff
@@ -1,11 +1,5 @@
 const path = require('path')
-const relativeServerPath = '../../servers/republik/'
-require('@orbiting/backend-modules-env').config(
-  path.join(__dirname, relativeServerPath, '.test.env')
-)
-require('@orbiting/backend-modules-env').config(
-  path.join(__dirname, relativeServerPath, '.env')
-)
+require('@orbiting/backend-modules-env').config()
```

Comment:
Isn't the `.test.env` needed here anymore?

```suggestion
require('@orbiting/backend-modules-env').config('.test.env')
```